mat-process-api/V1/Factories/ProcessDataFactory.cs

```diff
@@ -26,7 +28,7 @@ public static MatProcessData CreateProcessDataObject(BsonDocument bsonResult)
 
         public static MatProcessData CreateProcessDataObject(PostInitialProcessDocumentRequest requestObject) //Maps PostInitialDocumentRequest object to ProcessData domain object
         {
-            DateTime dateOfCreation = DateTime.UtcNow; // need this here because DateCreated and DateLastModified have to be equal
+            DateTime dateOfCreation = DateTime.Now; // need this here because DateCreated and DateLastModified have to be equal
```

Comment:
Is there a reason why this changed from ".Now" into ".UtcNow", and then changed back to ".Now"?